File: VB/PersonalOrganizer/Common/Utils/DbExtensions.vb
Imports System
Imports System.Collections
Imports System.Linq
Namespace Common.Utils
    Public Module DbExtensions
        <System.Runtime.CompilerServices.Extension> _
        Public Sub Load(ByVal source As IQueryable)
            Dim enumerator As IEnumerator = source.GetEnumerator()
            While enumerator.MoveNext()
            End While
        End Sub
    End Module
End Namespace

File: CS/PersonalOrganizer/Common/Utils/DbExtensions.cs
using System;
using System.Collections;
using System.Linq;

namespace PersonalOrganizer.Common.Utils {
    public static class DbExtensions {
        public static void Load(this IQueryable source) {
            IEnumerator enumerator = source.GetEnumerator();
            while(enumerator.MoveNext()) {
            }
        }
    }
}
